web/docker/40-envsubst-on-reearth-config.sh

```diff
@@ -2,8 +2,21 @@
 
 set -e
 
-# rewrite index.html to change title and favicon
+# rewrite index.html and published.html to change title and favicon
 _REEARTH_HTML_FILE="/usr/share/nginx/html/index.html"
+_REEARTH_PUBLISHED_HTML_FILE="/usr/share/nginx/html/published.html"
+
+# Rewrite title tag in index.html only if REEARTH_WEB_TITLE is set
+if [ -n "$REEARTH_WEB_TITLE" ]; then
+  sed -i -e "s|<title>.*</title>|<title>${REEARTH_WEB_TITLE}</title>|g" "$_REEARTH_HTML_FILE"
+  sed -i -e "s|<title>.*</title>|<title>${REEARTH_WEB_TITLE}</title>|g" "$_REEARTH_PUBLISHED_HTML_FILE"
+fi
+
+# Rewrite favicon in index.html and published.html only if REEARTH_WEB_FAVICON_URL is set
+if [ -n "$REEARTH_WEB_FAVICON_URL" ]; then
+  sed -i -e "s|<link rel=\"icon\" href=\"[^\"]*\" />|<link rel=\"icon\" href=\"${REEARTH_WEB_FAVICON_URL}\" />|g" "$_REEARTH_HTML_FILE"
+  sed -i -e "s|<link rel=\"icon\" href=\"[^\"]*\" />|<link rel=\"icon\" href=\"${REEARTH_WEB_FAVICON_URL}\" />|g" "$_REEARTH_HTML_FILE"
```

Comment:
Can you fix it?: published.html

Reply:
My bad, fixed it, thank you: https://github.com/reearth/reearth-visualizer/pull/1235/commits/6a848b314ec1d47b63663d54a5b4361b0e41b078